### ISSUE TITLE
mouse click cause exit on nvim-0.10

### ISSUE THREAD

**Issue by cxwx** [lewis6991/satellite.nvim#69]
**Describe the bug**
when I upgrade nvim-0.10 and try to use this plugin

**To Reproduce**
mouse left click on anywhere,
it cause exit nvim

**Expected behavior**
other plugins like hover.nvim

**Comment by lewis6991** [comment]
Works for me. You need to provide more information and reproduction steps.

**Comment by Servostar** [comment]
Had the same issue as described.
After updating neovim from version 0.9.5 to 0.10.0 and installing this plugin (with Lazy) neovim would exit when clicking the mouse.
No error messages, no notifications. Nothing.

Updating all other plugins I had installed (via Lazy) the problem is now gone and satellite works as expected.
Doesn't seem to have been an issue with satellite itself but rather a mixture or a new neovim version and old(er) plugins.

TLDR;
Maybe try updating. Resolved it for me.

**Comment by cxwx** [comment]
> Had the same issue as described. After updating neovim from version 0.9.5 to 0.10.0 and installing this plugin (with Lazy) neovim would exit when clicking the mouse. No error messages, no notifications. Nothing.
> 
> Updating all other plugins I had installed (via Lazy) the problem is now gone and satellite works as expected. Doesn't seem to have been an issue with satellite itself but rather a mixture or a new neovim version and old(er) plugins.
> 
> TLDR; Maybe try updating. Resolved it for me.

OK I'll try

**Comment by cxwx** [comment]
> Had the same issue as described. After updating neovim from version 0.9.5 to 0.10.0 and installing this plugin (with Lazy) neovim would exit when clicking the mouse. No error messages, no notifications. Nothing.
> 
> Updating all other plugins I had installed (via Lazy) the problem is now gone and satellite works as expected. Doesn't seem to have been an issue with satellite itself but rather a mixture or a new neovim version and old(er) plugins.
> 
> TLDR; Maybe try updating. Resolved it for me.

solved, but I'm not sure which mixture plugins
